src/main/java/com/google/sps/servlets/SentimentAnalysisServlet.java

```diff
@@ -17,22 +18,23 @@
 
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        String message = request.getParameter("user-message");
+        // This takes the request from the POST body
+        String message = request.getReader().lines().collect(Collectors.joining(System.lineSeparator()));
 
+        // builds our languageService and analyzes our message
         Document doc =
             Document.newBuilder().setContent(message).setType(Document.Type.PLAIN_TEXT).build();
         LanguageServiceClient languageService = LanguageServiceClient.create();
         Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+        
+        // gets the score from the sentiment analyzer
         float score = sentiment.getScore();
         languageService.close();
 
+
         // Output the sentiment score as HTML.
-        // A real project would probably store the score alongside the content.
-        response.setContentType("text/html;");
-        response.getWriter().println("<h1>Sentiment Analysis</h1>");
-        response.getWriter().println("<p>You entered: " + message + "</p>");
-        response.getWriter().println("<p>Sentiment analysis score: " + score + "</p>");
-        response.getWriter().println("<p><a href=\"/\">Back</a></p>");
+        response.setContentType("text/html");
+        response.getWriter().println(score);
```

Comment:
Maybe add some text here indicating to user what the score is?

Reply:
One of the reasons I choose to not add any text was to let the styling and displaying be handled in the HTML and/or script.js. I'm curious, however, if this is a bad practice with servlets and if the displaying of the sentiment should be handled here?

Reply:
Ok, thats fine in that case.